### DISCUSSION TITLE
achieving the Starstruck badge🌟

### DISCUSSION THREAD

**Discussion by drk1rd** [Schweinepriester/github-profile-achievements#428] [General]
Give this a star, and I'll do the same for you!
https://github.com/drk1rd/drk1rd

**Comment by HHNebula** [comment]
Done, bro. Do it for me, too :)
https://github.com/HHNebula/HHNebula

**Reply by drk1rd** [reply]
Done!

**Reply by callqh** [reply]
Done, bro. Do it for me, too :)
https://github.com/liuqh0609/note

**Comment by callqh** [comment]
> Give this a star, and I'll do the same for you! https://github.com/drk1rd/drk1rd

Done, bro. Do it for me, too :)
https://github.com/liuqh0609/note

**Reply by drk1rd** [reply]
Done!

**Reply by ArielMAJ** [reply]
@callqh I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by leilayesufu** [comment]
https://github.com/leilayesufu/quotes_generator

Kindly leave a star🥺

**Reply by montasim** [reply]
Done:
Do it for me, please.
https://github.com/montasim/montasim

**Reply by ArielMAJ** [reply]
@leilayesufu I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by montasim** [comment]
Done @drk1rd.
Do it for me, please.
https://github.com/montasim/montasim

**Comment by ArielMAJ** [comment]
@drk1rd I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
